### DISCUSSION TITLE
Outreachy Contribution Phase

### DISCUSSION THREAD

**Discussion by gbengaoluwadahunsi** [camicroscope/caMicroscope#680] [Q&A]
Hello, I am Gbenga,  I got accepted into the  Outreachy internship program, having read through the myriads of stellar projects, caMicroscope projects intrigue me the most as someone who has experience in Biomedicine and also a Software Developer. 

My request now is how to go about making my contributions.

Thanks.

**Comment by birm** [comment]
Hi! Thanks for reaching out! We've added a draft list here - https://github.com/camicroscope/GSOC/blob/master/ContributorTasks.md 
It's pretty new, so let us know if anything is unclear!

**Reply by gbengaoluwadahunsi** [reply]
Thank  you  Birm, I will go through it and revert.

**Reply by gbengaoluwadahunsi** [reply]
Hello @birm 

I love this project,  I have a pull  request for the dark mode for the landing page already, please check so  I can finish it up for other pages, again, how do I go about  adding suggestions to the the  UI of the  pages, will that be through a README file by a pull request?

**Reply by JUNNY12** [reply]
Hi @gbengaoluwadahunsi , it's Nice meeting you here. Please can you point out the repository you are working with to make changes to the UI, after cloning the distro repo, I can't seem to see any HTML, Css or Js file. I would be glad if you can assist.

**Reply by JUNNY12** [reply]
Hello @birm , I would love to contribute to the UI but I can't seems to understand the repository yet. I have managed to clone the distro repository and setup the development environment with docker and everything seems to be working well. But how do I make contribution to the UI?

**Reply by gbengaoluwadahunsi** [reply]
Hello Juwon, yes I had the same challenge initially with the Distro repo, I found out there were three repos that @birm  listed on the contribution page, the one that has the frontend development app folders in full is the "caMicroscope" repo, you can fork that and make your contributions.

**Reply by JUNNY12** [reply]
Thank you @gbengaoluwadahunsi , please if you don't mind can you share me the link here directly. I will greatly appreciate

**Reply by JUNNY12** [reply]
@gbengaoluwadahunsi Thanks for the heads up, I think I found the repository already

**Reply by OfficialOzioma** [reply]
Can someone help I am trying to set up this project, but getting some error on docker

**Reply by gbengaoluwadahunsi** [reply]
Hello @OfficialOzioma ,

Can you relay the error message you are getting, try and 
1. Install docker on your machine
2. I would suggest you build the website app on your machine before running it, I meant try 'docker-compose -f caMicroscope.yml build' before  'docker-compose -f caMicroscope.yml up'.

**Reply by JUNNY12** [reply]
Good day @gbengaoluwadahunsi , I need you to help me out with somethings. How were you able to load the frontend and make live changes to it

**Reply by gbengaoluwadahunsi** [reply]
Helllo @JUNNY12 , you find the frontend app in the caMicroscope viewer repo. 

I hope that helps.

**Comment by OfficialOzioma** [comment]
Hello, I am Ozioma, I got accepted into the Outreachy internship program, i will like contribute to the project

**Reply by nwanduka** [reply]
Welcome Ozioma. Glad to have you here
